openpype/hosts/unreal/plugins/load/load_geometrycache_abc.py

```diff
@@ -21,8 +26,11 @@ class PointCacheAlembicLoader(plugin.Loader):
     icon = "cube"
     color = "orange"
 
+    root = "/Game/Ayon/Assets"
```

Comment:
I would yank this out to some constant living in the unreal host code.

Reply:
Maybe it would be even better to make a setting to make the studio choose where to import the assets. What do you think?

Reply:
Changed to a constant for now!